nestkernel/parameter.h

```diff
@@ -49,9 +49,13 @@ class Parameter
 {
 public:
   /**
-   * Creates an Parameter with default values.
+   * Creates a Parameter with default values.
    */
-  Parameter() = default;
+  Parameter( bool is_spatial = false, bool returns_int_only = false )
+    : parameter_is_spatial_( is_spatial )
+    , parameter_returns_int_only_( returns_int_only )
```

Comment:
Why not drop `parameter_` from the field names here, it is rather obvious ...

Reply:
Done.